src/Container.js

```diff
@@ -39,11 +40,12 @@ export default class Container extends React.Component {
     return (
       <Grid>
         <Header data={this.state.posts[1].profile} />
-        {/*
-          this.state.posts.map((obj, index) =>
+        {
+          /*this.state.posts.map((obj, index) =>
```

Comment:
Kayaknya ini harus gini mas @adrianha, `<Container posts={this.state.posts} />` :D